recipes/datajoint/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "datajoint" %}
+{% set version = "0.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a35e8d87398b813aef95f4a1bfbaacae5cc905971d6cb99d3114ab2c0f35229b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

It doesn't really matter but `--no-deps` is no longer required:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

---

OK. The docs still suggest it: https://conda-forge.org/docs/maintainer/adding_pkgs.html, and I've been burned before by leaving it out (https://github.com/conda-forge/conda-forge.github.io/issues/684), so I think I'll leave it in if the recipe is still acceptable.